### ISSUE TITLE
mn

### ISSUE THREAD

**Issue by undrakh1103** [squidfunk/mkdocs-material#2710]
## Instructions

1. Check, if your language is already available: [here](https://bit.ly/33vFDD0)
2. If it isn't, please translate the labels on the right:

``` jinja
{% macro t(key) %}{{ {
  "language": "mn",
  "direction": "ltr",
  "clipboard.copy": "Хуулах",
  "clipboard.copied": "Санах ойд хуулах",
  "edit.link.title": "Хуудас засварлах",
  "footer.previous": "Өмнөх",
  "footer.next": "Дараах",
  "footer.title": "Хөл",
  "header.title": "Толгой",
  "meta.comments": "Сэтгэгдэл",
  "meta.source": "Эх үүсвэр",
  "nav.title": "Чиглүүлэгч",
  "search.config.lang": "mn",
  "search.config.pipeline": "trimmer, stopWordFilter",
  "search.config.separator": "[\s\-]+",
  "search.placeholder": "Хайлт",
  "search.reset": "Цэвэрлэх",
  "search.result.placeholder": "Хайлтын үгээ бичнэ үү",
  "search.result.none": "Таарц илэрсэнгүй",
  "search.result.one": "1 таарц илэрлээ",
  "search.result.other": "# Тохирох баримт бичиг",
  "search.result.more.one": "1 илүү хуудас байна",
  "search.result.more.other": "# илүү хуудас байна",
  "skip.link.title": "Агуулгыг алгасах",
  "source.link.title": "Хадгалах сан руу очих",
  "source.file.date.updated": "Сүүлийн шинэчлэлт",
  "source.file.date.created": "Үүсгэсэн",
  "tabs.title": "Табууд",
  "toc.title": "Агуулга"
}[key] }}{% endmacro %}
```

**Comment by squidfunk** [comment]
Thanks! Added in 093c67880.

**Comment by squidfunk** [comment]
Released as part of 7.1.6.
